xapian-bindings/java/SmokeTest.java

```diff
@@ -121,8 +121,20 @@ public static void main(String[] args) throws Exception {
 		System.exit(1);
 	    }
 	    Enquire enq = new Enquire(db);
-	    enq.setQuery(new Query(Query.OP_OR, "there", "is"));
+	    enq.setQuery(new Query("dog"));
 	    MSet mset = enq.getMSet(0, 10);
+	    // MSet size is 0
+	    if (mset.size() == 0) {
+	    for (MSetIterator.MatchDescriptor descriptor : mset) {
+	    System.err.println("Empty iterator entered invalid section (from MSet java iterator)");
```

Comment:
Okay, you need to configure sublime to use 4 character offsets for each level of nesting, and to prefer tabs over spaces. That way it'll use tabs as much as possible (at the left end of the line), and switch to spaces later (just before the text starts). The problem comes if there are tabs after spaces, in the leading whitespace, because some editors do different things with that, so it's confusing.

For instance, line 128 should really start with two tabs, which would make it appear further indented than the surrounding `if`.